### DISCUSSION TITLE
Tried to synchronously call a non-worklet function on the UI thread

### DISCUSSION THREAD

**Discussion by edvard-bjarnason** [software-mansion/react-native-reanimated#5539] [Q&A]
ReanimatedError: [Reanimated] Tried to synchronously call a non-worklet function on the UI thread.
See `https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting#tried-to-synchronously-call-a-non-worklet-function-on-the-ui-thread` for more details., js engine: reanimated

I get this error in the logs but I have no idea where the error is located. I understand the error and know how to probably fix it by adding 'worklet' to the function but I have no idea where to look. Is there a way to get a stack trace so the error can be located?

**Comment by MatiPl01** [comment]
Hey!

Don't you get the fragment of the source code where you call a non-worklet function? You should be able to see a screen similar to this, which shows the problematic line of code:

<img src="https://github.com/user-attachments/assets/30b748bf-c1e6-48ed-b097-9a10a6fe7aab" width="300" />

Could you try upgrading reanimated and see if the error location is now shown?
